### ISSUE TITLE
Login VK question

### ISSUE THREAD

**Issue by valentin-kushnarev** [so1ji/SamolazovProject#8]
Good evening! I'm just learning automation on Cypress. Help, please, deal with the problem of authorization VKontakte. After my test enters the password and clicks the "Login" button, the message "Invalid URL" appears in the form window. And there is no entry into the personal VK profile. Your code seems to be the same as mine. Maybe it's some other environment settings?

![image](https://github.com/DrGermanius/CypressTrainingProject/assets/121609422/0cefc3ec-ce40-4215-9934-d9c2fee20be6)

My code:
   ```
 it('Login', function (){
      cy.get('#index_email')
        .should('have.attr', 'placeholder', 'Телефон или почта')
        .realHover()
        .type(`${this.loginData.loginVk}`)
        .should('have.value',this.loginData.loginVk)
        .wait(5000)
      
      cy.get('button[type=submit]')
        .click()
        .wait(5000)
      
      cy.get('input[name=password]')
        .type(`${this.loginData.passwordVk}`)
        .should('have.value', this.loginData.passwordVk)
        .wait(3000)
      
      cy.get('button[type=submit]')
        .realHover('focus')
        .wait(5000)
        .realClick()
})
```

Please help in any way you can :)

**Comment by DrGermanius** [comment]
Hey there! I've tried using the following code: 
```       
cy.visit('https://vk.com')
cy.get('input[id="index_email"]').type(`+48221165200{enter}`)
cy.wait(2000)
cy.get('input[name="password"]').type(`passpass{enter}`)
```

It worked for me (I'm getting "wrong password").

I've faced some troubles running this script using Chrome, but it worked in Electron so you can try Electron as well.
Could you try to run these 4 lines above with your credentials?

<img width="742" alt="image" src="https://github.com/DrGermanius/CypressTrainingProject/assets/43386816/bfcf01e2-45c8-47f5-8fee-138738a78811">

**Comment by DrGermanius** [comment]
BTW it started to work in Chrome as well. Looks like temporary issue on my side

**Comment by valentin-kushnarev** [comment]
I get the same error in Chrome and Electron.


![image](https://github.com/DrGermanius/CypressTrainingProject/assets/121609422/ef906bce-29eb-4160-8280-d940a5bfb9dd)
"

**Comment by DrGermanius** [comment]
@valentin-kushnarev is it result of executing this code ?
```
cy.visit('https://vk.com')
cy.get('input[id="index_email"]').type(`+48221165200{enter}`)
cy.wait(2000)
cy.get('input[name="password"]').type(`passpass{enter}`)
```

**Comment by valentin-kushnarev** [comment]
Yes, with my credentials.

**Comment by DrGermanius** [comment]
Does it work without cypress under your credentials / ip ?

**Comment by valentin-kushnarev** [comment]
Yes, in a regular Сhrome browser with the same login / password everything works.
